src/js/lotto/util/createRandomLottoNumber.js

```diff
@@ -0,0 +1,27 @@
+export const createRandomLottoNumber = (count) => {
+  const lottoRandomNumbers = []
+  
+  const generateRandomNumbers = () => {
+    return  [ (Math.floor(Math.random() * 45) + 1),
+        ' ' + (Math.floor(Math.random() * 45) + 1),
+        ' ' + (Math.floor(Math.random() * 45) + 1),
+        ' ' + (Math.floor(Math.random() * 45) + 1),
+        ' ' + (Math.floor(Math.random() * 45) + 1),
+        ' ' + (Math.floor(Math.random() * 45) + 1),
+      ]
```

Comment:
```suggestion
    return  Array.from({ length: 6 }, v => Math.floor(Math.random() * 45) + 1).map(String);
```
이렇게 표현할 수 있답니다!
![image](https://user-images.githubusercontent.com/18749057/159166463-9f556118-69e0-4b3c-b44a-6977df63d377.png)

Reply:
반영했습니다!